public/enhanced-table-vis-controller.js

```diff
@@ -641,7 +641,9 @@ module.controller('EnhancedTableVisController', function ($scope, Private, confi
     }
     else {
       table.columns.forEach(column => {
-        column.aggConfig = aggConfigs.byId(column.aggConfig.id);
+        if (!column.aggConfig) {
+          column.aggConfig = aggConfigs.byId((column.aggConfig.parentId ? column.aggConfig.parentId : column.aggConfig.id));
+        }
```

Comment:
Well, your new commit looks wrong.
You check that "column.aggConfig" does not exist, but on the contrary, "column.aggConfig" must exist to use "column.aggConfig.id" to retrieve full aggConfig.
Actually, the initial aggConfig is not complete, so I add to retrieve the full one to replace initial aggConfig.

By the way, I have not pushed the code yet, but I fixed #79 issue with this code:
`column.aggConfig = aggConfigs.byId(column.aggConfig.id) || column.aggConfig;`

It should also fix #81 issue.

Reply:
Yep, you are right.

So this should be better:

`column.aggConfig = aggConfigs.byId(column.aggConfig.id) || aggConfigs.byId(column.aggConfig.parentId) || column.aggConfig;`

Reply:
Yes

Reply:
Do you want me to change this and push the commit?